docker-compose.yml

```diff
@@ -17,4 +17,31 @@ services:
       --coinbase-password $${COINBASE_PASSWORD} \
       --price-feed $${ETH_DAI_PRICE_FEED?:} \
       --pair ETH-DAI \
-      --config /opt/maker/config/$${COINBASE_ETH_DAI_BANDS?:}"
\ No newline at end of file
+      --config /opt/maker/config/$${COINBASE_ETH_DAI_BANDS?:}"
+
+  parity:
```

Comment:
I see the ganache image is used for unit testing.  Why do we need the MCD dockerized Parity testchain?